bolt-sidecar/src/driver.rs

```diff
@@ -74,17 +96,24 @@ impl<C: StateFetcher, BLS: SignerBLS, ECDSA: SignerECDSA> SidecarDriver<C, BLS,
         let (api_events_tx, api_events_rx) = mpsc::channel(1024);
         CommitmentsApiServer::new(api_addr).run(api_events_tx).await;
 
+        // TODO: this can be replaced with ethereum_consensus::clock::from_system_time()
+        // but using beacon node events is easier to work on a custom devnet for now
+        // (as we don't need to specify genesis time and slot duration)
+        let head_tracker = HeadTracker::start(beacon_client.clone());
+
+        // Initialize the consensus clock.
+        let consensus_clock = from_system_time(
```

Comment:
can we leave the package name for clarity here? e.g. `clock::from_system_time()` otherwise it's a bit unclear imo